### DISCUSSION TITLE
Calculate The dice coefficient for testing dataset

### DISCUSSION THREAD

**Discussion by shathaa1983** [TorchIO-project/torchio#990] [General]
Hi,

I am trying to calculate the dice coefficient for the model applied on different dataset. But I am unable to do that.

Could you please help with this?

Thanks

**Comment by fepegar** [comment]
Hi, @shathaa1983, This is out of the scope of TorchIO.

I would import MONAI's [`DiceLoss`](https://docs.monai.io/en/stable/losses.html#diceloss).

**Reply by shathaa1983** [reply]
Thank you for the reply.
So in the tutorial using google colab, you applied the Unet model to do brain segmentation. I tried to know the total parameters for the Unet model, and I got it (total_params= 246156). Is this number correct?
